packages/terra-list/CHANGELOG.md

```diff
@@ -7,6 +7,11 @@
 * Changed
   * Sections and subsections updated to add programmatic context to communicate hierarchal relationships
 
+* Added
+  * Prop `ariaDescribedBy` - Associates a list to an element or elements that convey information about the list and how to interact with it.
+  * Prop `ariaDescription` - Conveys information about a list and how to interact with it.
+  * Prop `ariaDetails` - Associates a list to an element or elements that convey information about the list and how to interact with it.
```

Comment:
Would it be `Associates a list with an element` be more accurate? Or alternatively, `Maps a list to an element`?

Reply:
Updated wording to 'with' [250e017](https://github.com/cerner/terra-core/pull/3688/commits/250e0179b89df8feb46bc63796388b7619fdbef6)